core/pom.xml

```diff
@@ -359,6 +359,12 @@
       <artifactId>py4j</artifactId>
       <version>0.8.2.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.intel.chimera</groupId>
+      <artifactId>chimera</artifactId>
+      <version>0.0.1</version>
```

Comment:
So, how stable is this project? It seems, from the github page, to be a re-packaging of the Hadoop code with maybe some more bells & whistles, but the version number doesn't scream stability...